content/1.7/back-office/shop-parameters/contact/stores/add-edit stores.md

```diff
@@ -0,0 +1,97 @@
+If the save button is pressed and if this field is empty, the following error message is displayed: "The field address1 is required at least in _default language_"
+
+3) Address (2)
+
+This input field allows to fill the second address line of the store. This field can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
+
+4) Zip/Postal code
+
+This input field allows to fill the zip/postal code of the store. The field is not marked as mandatory, but if the field is empty, and trying to save, shows the message: "A Zip/Postal code is required.". Also the postcode can't be too long, if it's too long, and trying to save shows the message: "The postcode field is too long (12 chars max).", also if the format isn't correct, when saving shows error: "The Zip/Postal code is invalid. It must be entered as follows: 00000"
+
+5) City
+
+This input field allows to fill the city of the store and is mandatory.
+If the save button is pressed and if this field is empty, the following error message is displayed: "The city field is required."
+
+6) Country
+
+This drop-down allows to select the country of the store and is mandatory. While creating a new store, the default country (selected in International > Localization > Configuration) is selected by default.
+
+7) State (only for the countries that have states, as defined in International > Locations > States.)
+
+This drop-down allows to select the state of the store and is mandatory. If no state is selected, when trying to save, shows: "An address located in a country containing states must have a state selected."
+
+8) Latitude/Longtitude
+
+There are 2 separate input fields for the latitude and longitude of the store, they are both mandatory.
+Only numbers are allowed for these fields. When trying to input characters that are not some numbers, an error message is displayed: "Latitude and longitude are required." The numbers in the fields are limited to 13, if there are more numbers than allowed, an error message is displayed: "The latitude field is too long (13 chars max)." and/or "The longitude field is too long (13 chars max)."
+When hovered on the writing - Latitude / Longitude, shows the following: "Store coordinates (e.g. 45.265469/-47.226478).
+
+9) Phone
+
+This input field allows to fill the phone number of the store.
+
+10) Fax
+
+This input field allows to fill the fax number of the store.
+
+11) Email address
+
+This input field allows to fill the email address of the store.
+
+12) Note
+
+This input field allows to fill a note for the store. This field can be translated, a drop-down list is shown, translates the field to a selected language which are added in the shop.
```

Comment:
```suggestion
This input field allows to fill a note for the store. It can be translated: When several languages are installed, a drop-down next to the field is displayed. It allows choosing in which language the field is displayed and customizing it according to the language. Disabled languages are displayed in the drop-down.
```